.github/workflows/MainDistributionPipeline.yml

```diff
@@ -0,0 +1,31 @@
+#
+# This workflow calls the main distribution pipeline from DuckDB to build, test and (optionally) release the extension
+#
+name: Main Extension Distribution Pipeline
+on:
+  push:
+  pull_request:
```

Comment:
Should this run on pull requests as well?

Reply:
I think so? I removed most of the other jobs so without it, we would only test on linux gcc4 for PRs. I could add a `needs` to make it depend on the `linux-tests-postgres` to reduce ci time for broken builds?